### PR TITLE
Changes in gg_miss_ functions

### PR DESCRIPTION
**Pull request by ColinFay**
## Description

You can now do : 

```
gg_miss_var(airquality) + theme_bw() 
gg_miss_case(airquality) + labs(x = "Number of Cases")
gg_miss_var(airquality) + labs(y = "Look at all the missing ones") + theme_linedraw()
gg_miss_which(airquality) + theme(panel.background = element_rect(fill = "grey"))
```
These changes don't break old code. The upgrade allows you to use these functions as you used to. They have a default theme setting, but you can simply override these settings by providing ggplot functions. You can also add other geom if you wish. Basically you can do anything you can do with a ggplot :)  

## Related Issue
 
I couldn't get the `gg_miss_span` to work. I couldn't try this one, yet I'm pretty confident this fourth function works. 

Best, 

Colin

### CODE REVIEW

**Comment by njtierney** [comment]
✨ 
Thank you so much!

I'll have a review of this tonight, really appreciate your help, @ColinFay :)

**Comment by njtierney** [comment]
Just quickly, can you bump the dev version number by 100 and add a news bullet describing these awesome changes you made?

**Comment by ColinFay** [comment]
Done ;)

File: R/plotting.R [hunk 1]
```diff
@@ -118,6 +118,36 @@ gg_miss_which <- function(x){
...
+  cond <- enquo(fac)
+  tab <- x %>%
+    group_by(!!cond) %>%
+    do(miss_var_summary(.))
+  names(tab)[1] <- "Factor"
+  ggobject <- tab %>%
+    ggplot(aes(Factor, variable, fill = n_missing)) +
+    geom_tile()
+  return(ggobject)
+}
```
Comment (njtierney) [inline R/plotting.R]: So this is totally awesome! great job.

**Comment by njtierney** [comment]
Thank you!! This is really cool. It seems that gg_miss_fac works for me

**Comment by ColinFay** [comment]
The function works, but I'm not happy with the way it does. I used a little hack with names() that I'm not happy with. 
I'll work on a cleaner version of the code soon :)

**Pull request merged**